### PR TITLE
Add ability to scale the overview map compared to the view's scale

### PR DESCRIPTION
**Pull request by colinanderson**
Adds a property that scales the overview map relative to the main view.

### CODE REVIEW

**Comment by jenmerritt** [comment]
@colinanderson Additional property looks good. Couple of points as discussed:
- latest `master` can be merged in with the version updates etc. from earlier so it builds
- javadoc to add
- scales well with MapView but not with SceneView - should we match the behaviour?

**Comment by colinanderson** [comment]
@jenmerritt doc has been added and the behaviour has been updated to match the design. I've made a few other small changes like using `ColorUtil.colorToArgb` to define the colours.

File: src/main/java/com/esri/arcgisruntime/toolkit/OverviewMap.java [hunk 1]
```diff
@@ -191,4 +197,33 @@ public void setSymbol(Symbol symbol) {
   public SimpleObjectProperty<Symbol> symbolProperty() {
     return symbolProperty;
   }
+
+  /**
+   * A property containing The amount to scale the OverviewMap compared to the geoView. The default is 25.0.
```
Comment (jenmerritt) [inline src/main/java/com/esri/arcgisruntime/toolkit/OverviewMap.java:202]:
```suggestion
   * A property containing the amount to scale the OverviewMap compared to the geoView. The default is 25.0.
```

**Review by jenmerritt — APPROVED** [review]
Looks good to me and worked well when I tested. One tiny comment correction.

**Comment by colinanderson** [comment]
@sclaridge can you do a second review please?

**Review by sclaridge — APPROVED** [review]
This looks good to me!

**Pull request merged**